### PR TITLE
Removed redundant code and other refactoring

### PR DESCRIPTION
**Pull request by bharathkalyans**
1. Worked on Drifty_CLI, CreateLogs, FileLoader and ProgressBarThread.
2. Created package for the utility functions which were extracted from Drifty_CLI file.
3. Moved the HardCoded strings to DriftyConstant file.
4. Created Singletons for the Scanner class and CreateLogs file.

### CODE REVIEW

**Comment by bharathkalyans** [comment]
Hi @SaptarshiSarkar12, I accidentally closed yesterdays PR, so had to open a new one. I  have made the changes as requested.

**Comment by SaptarshiSarkar12** [comment]
@bharathkalyans Please create a GitHub issue describing the changes that you want to make and link that with this PR.

**Comment by SaptarshiSarkar12** [comment]
I shall review it tomorrow. @bharathkalyans . Let me know when you have made the issue.

**Comment by bharathkalyans** [comment]
- This PR resolves #194

**Comment by bharathkalyans** [comment]
Created the issue and linked to it.

**Comment by SaptarshiSarkar12** [comment]
> Created the issue and linked to it.

Great @bharathkalyans :+1: ! Shall review it tomorrow.

**Comment by bharathkalyans** [comment]
The codebase still needs a lot of refactoring, renaming of the Class variables and stuff like that.Will look into it I once I am free!

**Review by SaptarshiSarkar12 — CHANGES_REQUESTED** [review]
@bharathkalyans Please make the requested changes and then, I shall approve this PR.

**Comment by SaptarshiSarkar12** [comment]
> The codebase still needs a lot of refactoring, renaming of the Class variables and stuff like that.Will look into it I once I am free!

That's great. You may do so.

File: src/CheckDirectory.java [hunk 1]
```diff
@@ -4,25 +4,29 @@
...
 
+import static constants.DriftyConstants.*;
+
 /**
  * This class checks if a directory exists or not. if it doesn't, the directory is created.
  */
 class CheckDirectory {
     /**
      * This constructor creates the directory if it does not exist.
+     *
```
[the remark this answers is not in the document]
Comment (bharathkalyans) [reply src/CheckDirectory.java:15]: Do you want me to remove these extra comments ?

[the remark this answers is not in the document]
Comment (bharathkalyans) [reply src/CheckDirectory.java:15]: Do you want me to remove these extra comments ?

[the remark this answers is not in the document]
Comment (bharathkalyans) [reply src/CheckDirectory.java:15]: Can you be a bit more specific!

[the remark this answers is not in the document]
Comment (SaptarshiSarkar12) [reply src/CheckDirectory.java:15]: Yes @bharathkalyans .

[the remark this answers is not in the document]
Comment (SaptarshiSarkar12) [reply src/CheckDirectory.java:15]: You can directly click on `commit suggestion` button if there is a monor change.

**Comment by bharathkalyans** [comment]
> @bharathkalyans Please make the requested changes and then, I shall approve this PR.

Done with the changes 👍🏼

**Comment by SaptarshiSarkar12** [comment]
> > @bharathkalyans Please make the requested changes and then, I shall approve this PR.
> 
> Done with the changes 👍🏼

Great!

**Review by SaptarshiSarkar12 — APPROVED** [review]
Looks good to merge :+1: 
Thanks for contributing :rocket: :rocket: :rocket: !!
**@bharathkalyans If you liked contributing to the project, please leave a star :star: . You may join our Discord server for the project's discussions - https://discord.gg/DeT4jXPfkG**

**Pull request merged**